src/api/usersRouter.ts

```diff
@@ -62,15 +62,33 @@ usersRouter.post('/:userId/resendInvitation', async (req: UserRequest, res) => {
       return res.sendStatus(404);
     }
     if (resultLength > 1) {
-        errorLogger.error(
-          `Multiple results received when loading user entry for ${req.user?.email}`
-        );
+      errorLogger.error(`Multiple results received when loading user entry for ${req.user?.email}`);
       return res.sendStatus(500);
     }
 
     logger.info(`Resending invitation email for ${req.user?.email}, keycloak ID ${user[0].id}`);
     await sendInviteEmail(kcAdminClient, user[0]);
     return res.sendStatus(200);
-});
+  });
+
+  usersRouter.delete('/:userId', async (req: UserRequest, res) => {
+    const { user } = req;
+    await user!.$query().delete();
```

Comment:
I think this should have some protection to prevent getting an account into a bad state - probably block a user from deleting themselves. That way a participant can never completely lock themselves out.